StripePaymentSheet/StripePaymentSheet/Source/Categories/String+Localized.swift

```diff
@@ -311,4 +311,11 @@ extension String.Localized {
             "Title shown above a view allowing the customer to save a card."
         )
     }
+
+    static var add_us_bank_account: String {
+        STPLocalizedString(
+            "Add US bank account",
+            "Title shown above a view allowing the customer to save a US bank account."
```

Comment:
```suggestion
            "Title shown above a view allowing the customer to add a US bank account."
```
Let's not use save, since the customer isn't necessarily saving their bank account.